decentraland/AppNavigator.js

```diff
@@ -61,29 +91,19 @@ const MyAssetsNavigator = createStackNavigator(
   }
 );
 
-const MainDrawerNavigator = createDrawerNavigator(
-  {
-    AssetsForSale: {
-      screen: AssetsForSaleNavigator,
-      navigationOptions: {
-        drawerLabel: "Assets for sale",
-      },
-    },
-    MyAssets: {
-      screen: MyAssetsNavigator,
-      navigationOptions: {
-        drawerLabel: "My Assets",
-      },
+const MainDrawerNavigator = createDrawerNavigator({
+  AssetsForSale: {
+    screen: AssetsForSaleNavigator,
+    navigationOptions: {
+      drawerLabel: "Assets for sale",
     },
   },
-  {
-    drawerOpenRoute: "DrawerOpen",
-    drawerCloseRoute: "DrawerClose",
-    drawerToggleRoute: "DrawerToggle",
+  MyAssets: {
+    screen: MyAssetsNavigator,
     navigationOptions: {
-      drawerLockMode: "locked-closed",
```

Comment:
Is this option the default? Or do we want to default to whatever else the default is now that the drawer menu isn't always shown?

Reply:
Yes, they are default values (I've forget to delete these before creating the PR). These options aren't mentioned on `react-navigation` doc and seem that are used only for customization needs.
Refs:
https://reactnavigation.org/docs/en/drawer-navigator.html
https://github.com/react-navigation/react-navigation/issues/3266#issue-288153811

Reply:
It looks like drawerLockMode is mentioned in their docs, anyway

Reply:
Good to know they’re defaults - thanks for looking into this. Resolving!